homework-1/task1-2.js

```diff
@@ -0,0 +1,85 @@
+//
+// TASK 1.2
+//
+
+
+// -------------------------------------------
+// Test 1
+// Use command line to convert CSV to JSON
+
+// csvtojson csv/nodejs-hw1-ex1.csv > json/nodejs-hw1-ex2.json
+
+// END: Test 1
+// -------------------------------------------
+
+
+
+// -------------------------------------------
+// Test 3
+// CSV file is loaded fully into the RAM
+
+// const fs = require('fs');
+import fs from 'fs';
+// const csvtojson =require('csvtojson');
+import csvtojson from 'csvtojson';
+
+const csvfilepath = "csv/nodejs-hw1-ex1.csv"
+
+csvtojson()
```

Comment:
it's still needed to implement reading, transforming, and writing this file by using streams